lib/cppcheck.cpp

```diff
@@ -1143,6 +1143,10 @@ void CppCheck::executeRules(const std::string &tokenlist, const Tokenizer &token
         if (rule.pattern.empty() || rule.id.empty() || rule.severity == Severity::none || rule.tokenlist != tokenlist)
             continue;
 
+        if (!mSettings.quiet) {
+            std::cout << "Processing rule: " << rule.pattern << std::endl;
```

Comment:
I will let you decide about this.. if such message should be shown or not. However such messages should not be reported with `std::cout' because then they will not be seen in the GUI. Please use reportOut instead.